src/resolver.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright 2013-2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use Cancellable;
+use Error;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use std::ptr;
+use glib_ffi;
+use gobject_ffi;
+use Resolver;
+use InetAddress;
+use SrvTarget;
+
+pub trait ResolverExtManual {
+    fn lookup_by_address_async<'a, P: Into<Option<&'a Cancellable>>, Q: FnOnce(Result<String, Error>) + Send + 'static>(&self, address: &InetAddress, cancellable: P, callback: Q);
+
+    fn lookup_by_name_async<'a, P: Into<Option<&'a Cancellable>>, Q: FnOnce(Result<Vec<InetAddress>, Error>) + Send + 'static>(&self, hostname: &str, cancellable: P, callback: Q);
+
+    #[cfg(any(feature = "v2_34", feature = "dox"))]
+    fn lookup_records_async<'a, P: Into<Option<&'a Cancellable>>, Q: FnOnce(Result<Vec<glib::Variant>, Error>) + Send + 'static>(&self, rrname: &str, record_type: ResolverRecordType, cancellable: P, callback: Q);
```

Comment:
No type this function
```
#[cfg(any(feature = "v2_34", feature = "dox"))]
use ResolverRecordType;
```

Reply:
also `#[cfg(any(feature = "v2_34", feature = "dox"))]use glib;` needed or `glib::Variant` with fixing type

Reply:
Thanks, fixed